### PR TITLE
Ins 3300 cbor to proto

### PR DESCRIPTION
**Pull request by egorikas**
**- What I did**
Some of ledger entities had used cbor for serializing. I moved them to protobuf

### CODE REVIEW

File: ledger/heavy/executor/initial_state_keeper.go [hunk 1]
```diff
@@ -95,21 +96,18 @@ func (isk *InitialStateKeeper) Start(ctx context.Context) error {
 }
 
 func (isk *InitialStateKeeper) prepareDrops(ctx context.Context) {
-	logger := inslogger.FromContext(ctx)
 	for _, jetID := range isk.jetAccessor.All(ctx, isk.syncPulse) {
 		dr, err := isk.dropStorage.ForPulse(ctx, jetID, isk.syncPulse)
 		if err != nil {
-			logger.Fatal("No drop found for pulse: ", isk.syncPulse.String())
+			panic(fmt.Sprint("No drop found for pulse: ", isk.syncPulse.String()))
```
Comment (andreyromancev) [inline ledger/heavy/executor/initial_state_keeper.go]: logger.Panic?

File: ledger/heavy/executor/initial_state_keeper.go [hunk 2]
```diff
@@ -158,14 +156,14 @@ func (isk *InitialStateKeeper) Get(ctx context.Context, lightExecutor insolar.Re
...
-	drops := make([][]byte, 0)
+	drops := make([]drop.Drop, 0)
 	indexes := make([]record.Index, 0)
 
 	logger.Debugf("[ InitialStateKeeper ] Getting drops for: %s in pulse: %s", lightExecutor.String(), pulse.String())
 	for id, jetDrop := range isk.jetDrops {
 		light, err := isk.jetCoordinator.LightExecutorForJet(ctx, insolar.ID(id), pulse)
 		if err != nil {
-			logger.Fatal("No drop found for pulse ", isk.syncPulse.String())
+			panic(fmt.Sprint("No drop found for pulse ", isk.syncPulse.String()))
```
Comment (andreyromancev) [inline ledger/heavy/executor/initial_state_keeper.go]: logger.Panic?

File: ledger/heavy/executor/initial_state_keeper.go (hunk 1 above)
Comment (LinnikD) [inline ledger/heavy/executor/initial_state_keeper.go]: change it back pls

File: logicrunner/builtin/contract/noderecord/noderecord.go [hunk 3]
```diff
@@ -58,16 +58,12 @@ func NewNodeRecord(publicKey string, roleStr string) (*NodeRecord, error) {
...
 
-var INSATTR_GetNodeInfo_API = true
-
 // GetNodeInfo returns RecordInfo.
 // ins:immutable
 func (nr NodeRecord) GetNodeInfo() (RecordInfo, error) {
 	return nr.Record, nil
 }
 
-var INSATTR_GetPublicKey_API = true
```
Comment (LinnikD) [inline logicrunner/builtin/contract/noderecord/noderecord.go]: INSATTR_GetPublicKey_API this is nesussary for our API. Add nolint if your linter wants to delete it

File: ledger/heavy/executor/initial_state_keeper.go [hunk 4]
```diff
@@ -95,21 +96,18 @@ func (isk *InitialStateKeeper) Start(ctx context.Context) error {
 }
 
 func (isk *InitialStateKeeper) prepareDrops(ctx context.Context) {
-	logger := inslogger.FromContext(ctx)
 	for _, jetID := range isk.jetAccessor.All(ctx, isk.syncPulse) {
 		dr, err := isk.dropStorage.ForPulse(ctx, jetID, isk.syncPulse)
 		if err != nil {
-			logger.Fatal("No drop found for pulse: ", isk.syncPulse.String())
+			inslogger.FromContext(ctx).Fatal(fmt.Sprint("No drop found for pulse: ", isk.syncPulse.String()))
```
Comment (andreyromancev) [inline ledger/heavy/executor/initial_state_keeper.go]: `fmt.Sprint`

File: ledger/heavy/executor/initial_state_keeper.go [hunk 5]
```diff
@@ -158,14 +156,14 @@ func (isk *InitialStateKeeper) Get(ctx context.Context, lightExecutor insolar.Re
...
-	drops := make([][]byte, 0)
+	drops := make([]drop.Drop, 0)
 	indexes := make([]record.Index, 0)
 
 	logger.Debugf("[ InitialStateKeeper ] Getting drops for: %s in pulse: %s", lightExecutor.String(), pulse.String())
 	for id, jetDrop := range isk.jetDrops {
 		light, err := isk.jetCoordinator.LightExecutorForJet(ctx, insolar.ID(id), pulse)
 		if err != nil {
-			logger.Fatal("No drop found for pulse ", isk.syncPulse.String())
+			logger.Fatal(fmt.Sprint("No drop found for pulse ", isk.syncPulse.String()))
```
Comment (andreyromancev) [inline ledger/heavy/executor/initial_state_keeper.go]: `fmt.Sprint`

File: ledger/heavy/executor/jetkeeper.go [hunk 6]
```diff
@@ -139,7 +132,11 @@ func (j *jetInfo) isConfirmed(checkBackup bool) bool {
 		return false
 	}
 
-	if !*j.Split {
+	if !j.IsSplitSet {
+		panic("unexpected situation")
```
Comment (andreyromancev) [inline ledger/heavy/executor/jetkeeper.go]: @ivanshibitov is it ok to just return `false` here?

File: ledger/heavy/executor/jetkeeper.go [hunk 7]
```diff
@@ -344,7 +341,10 @@ func (jk *DBJetKeeper) getTopSyncJets() ([]insolar.JetID, error) {
 	}
 
 	for _, ji := range jets {
-		if *ji.Split {
+		if !ji.IsSplitSet {
+			panic("unexpected situation")
```
Comment (andreyromancev) [inline ledger/heavy/executor/jetkeeper.go]: @ivanshibitov is it ok to just treat it as `false` here?

**Pull request merged**